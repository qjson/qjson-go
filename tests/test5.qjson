1970-01-01T00:00:00+00:00:test
t0: [1970-01-01T, 
    1970-01-01T00:00, 
    1970-01-01T00:00Z,
    1970-01-01T00:00:00,
    1970-01-01T00:00:00Z,
    1970-01-01T00:00:00.000,
    1970-01-01T00:00:00.000Z,
    1970-01-01T00:00:00.000000,
    1970-01-01T00:00:00.000000Z,
    1970-01-01T00:00:00+00:00,
    1970-01-01T00:00:00-00:00,
    1970-01-01T00:00:00.000+00:00,
    1970-01-01T00:00:00.000-00:00,
    1970-01-01T00:00:00.000000+00:00,
    1970-01-01T00:00:00.000000-00:00]
t1: [1970-01-02T, 
    1970-01-01T00:01, 
    1970-01-01T01:00Z,
    1970-01-01T01:01:01,
    1970-01-01T00:02:02Z,
    1970-01-01T00:01:01.123,
    1970-01-01T01:01:01.123456,
    1970-01-01T00:00:01-01:01,
    1970-01-01T01:00:00+01:00]

