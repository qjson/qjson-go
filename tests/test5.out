{"1970-01-01T00:00:00+00:00":"test","t0":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"t1":[86400,60,3600,3661,122,61.123,3661.123456,3661,0]}
